ch5-mkdir/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
3.5 is the default one in Ubuntu?

Reply:
No, it's what Qt Creator gives by default in the CMakeLists.txt.
Do you think that there's a reason to change it?